home_work_9/index.js

```diff
@@ -0,0 +1,173 @@
+const validateValue = (value) => {
+  if (value === undefined || value === null) {
+    return;
+  }
+  return value;
+};
+
+const EMPTY_VALUE_MESSAGE = "This value is not specified";
+
+function addSpaseInArray(array) {
+  if (validateValue(array) || !array?.length === 0)
```

Comment:
As an option, it can be simplified 
```suggestion
  if (validateValue(array) || !array?.length)
```